features/79-structured-logging/api.md

```diff
@@ -0,0 +1,146 @@
+# Structured Logging
+
+## Feature Overview
+
+* Support production logs filtering
+* Show logs with color to make different kinds of log easier to see
+
+## Requirements
+
+* Support different filtering to make developer easier to discover the logs they want
+    1. filter by log level
+        - warning
+        - error
+        - info
+        - debug
+    1. filter by component
+        - Server
+        - Chat plugin
+        - CMS plugin
+        - Forgot password plugin
+        - SSO plugin
```

Comment:
I think a component is `Cloud Code Functions`, which include user-written logs.

Reply:
Sure, do you think it is good if we use `Cloud Code Functions`  as default value of component for log function? So when user call the log function they don't have to put so many argument.